src/styles/booklist.css

```diff
@@ -0,0 +1,23 @@
+.booksDetailsBox {
+  background: #1d1d1d;
+}
+
+.listDate {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 0;
+  height: 10vh;
+  font-size: 3vh;
+  width: auto;
+  font-weight: 900;
+  background: #3a5b99;
+}
+
+.loading {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100vh;
+}
```

Comment:
As part of the project requirements, kindly make sure that your design follows the design guidelines in terms of the layout (composition and space between elements). There are some slight differences I see between your design and that of the actual design as shown below: 

- [ ] The position of the image covers the top part with the white small arrow in a circle at the top right corner with the text below them
- [ ] The chequered nature of the books' background also follows a certain pattern. The first row starts with light background followed by darker background whiles the second row starts with dark background followed by lighter background. Each column also alternates in the same order as shown below. :point_down: 


### Your design 

![Screenshot from 2023-03-16 23-54-59](https://user-images.githubusercontent.com/65233860/225780349-c605eba6-be47-4aae-a9b7-28a806780745.png)



### Actual design 

![Screenshot from 2023-03-16 23-55-17](https://user-images.githubusercontent.com/65233860/225780424-d6fcab8a-56c9-49c6-9344-03a7b78fd41e.png)